ui-tests/src/test/java/org/openmrs/reference/page/EditVisitNotePage.java

```diff
@@ -10,12 +10,24 @@
 package org.openmrs.reference.page;
 
 import org.openmrs.uitestframework.page.Page;
+import org.openqa.selenium.By;
 
 public class EditVisitNotePage extends VisitNotePage {
-
+    private static final By NOTE = By.id("w10");
+    private static String  note = "this is edited note";
+    private static final By REMOVE_DIAGNOSIS = By.cssSelector(".icon-remove.delete-item");
     public EditVisitNotePage(Page page) {
```

Comment:
Should we not add an empty line to separate declared variables and methods  to ensure quality within our code base?

Reply:
Added a new line , though to me as fur as i have learnt openmrs code especially with in openmrs core, this is no where implemented , though am seeing its getting growing with in current qa work, so i dont know whether we  can continue to implement it in any other code with in openmrs cc @dkayiwa  @ibacher

Reply:
I object neither!
However, I want to believe that embracing software standards in our code base is one our role as QA Team to ensure our code base aligns with international software standards. I do not mean that our current code-base does not align with the standards, but there might be some oversights within our code base that we need to work on within our community and to me this sounds to make a difference why QA Team was embraced in the community.

@sherrif10 in reference to my suggestion [here](https://github.com/openmrs/openmrs-distro-referenceapplication/pull/450/files#r679962005), I gathered some artifacts on this opinion [here](https://github.com/openmrs/openmrs-distro-referenceapplication/pull/476#issuecomment-870471982)